doc/user_guide/scale_resolve.rst

```diff
@@ -43,4 +43,65 @@ This default can be changed by setting the scale resolution for the color to
        base.encode(color='Cylinders:O')
    ).resolve_scale(
        color='independent'
-   )
\ No newline at end of file
+   )
+
+Dual Y Axis
+~~~~~~~~~~~
+
+A common technique for combining chart containing different measures is using a
+dual y axis. There are two strategies to achieve this result using altair. The
+first is to manually specify the mark color and associated axis title color of
+each layer. 
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.cars()
+
+    base = alt.Chart(source).encode(
+            alt.X('year(Year):T')
+    )
+
+    line_A = base.mark_line(color='#5276A7').encode(
+        alt.Y('average(Horsepower):Q', axis=alt.Axis(titleColor='#5276A7'))
+    )
+
+    line_B = base.mark_line(color='#F18727').encode(
+        alt.Y('average(Miles_per_Gallon):Q', axis=alt.Axis(titleColor='#F18727'))
+    )
+
+    alt.layer(line_A, line_B).resolve_scale(y='independent')
+
+In this case the axis colors act as a pseudo-legend. Alternatively if you want
+a legend some transformations must be applied. Legends are only created in
+Vega-Lite to represent an encoding.
```

Comment:
It would be useful to link to the fold and filter transform docs here: ``:ref:`user-guide-filter-transform` `` and ``:ref:`user-guide-fold-transform` ``